ncbi_genome_download/core.py

```diff
@@ -105,6 +106,8 @@ def argument_parser(version=None):
                         help='Dump all files right into the output folder without creating any subfolders.')
     parser.add_argument('-H', '--human-readable', dest='human_readable', action='store_true',
                         help='Create links in human-readable hierarchy (might fail on Windows)')
+    parser.add_argument('-pb', '--progress-bar', dest='progress_bar', action='store_false',
```

Comment:
I'm not a big fan of two-letter "short" abbreviations. Maybe `-P` would be a better choice.